test/com/core/tictactoe/HumanPlayerTests.java

```diff
@@ -28,7 +28,7 @@ void returnsAMove() {
         String[] fakeUserInputs = {"1"};
         StubbCommandLineUI stubbCommandLineUI = new StubbCommandLineUI(new PrintStream(output), System.in, fakeUserInputs);
```

Comment:
I think Stub only has one 'b', and where you use the word 'UI', I think it's better practice to capitalise UI like the other words – so you could write `StubCommandLineUi stubCommandLineUi`

Reply:
Done.